runtime/tutor/en/vim-01-beginner.tutor

```diff
@@ -20,8 +20,7 @@ pressing [<Esc>](<Esc>) and then [u](u) will undo the latest change.
 This tutorial is interactive, and there are a few things you should know.
 - Type [<Enter>](<Enter>) on links [like this](holy-grail    ) to open the linked help section.
 - Or simply type [K](K) on any word to find its documentation!
-- Sometimes you will be required to modify text like
-this here
```

Comment:
This seems to be intentional but `this here` on a separate line is awkward.
I'd expect this line to have linebreaks before and after this line and indented to be make it standout.